huaweicloud/services/dli/resource_huaweicloud_dli_queue.go

```diff
@@ -357,6 +365,20 @@ func resourceDliQueueUpdate(ctx context.Context, d *schema.ResourceData, meta in
 		}
 	}
 
+	oldVal, newVal := d.GetChange("elastic_resource_pool_name")
```

Comment:
it should be wrapped with d.HasChange("elastic_resource_pool_name")

Reply:
fixed